### PR TITLE
Multi map

### PR DESCRIPTION
**Pull request by Sindex42**

### CODE REVIEW

**Review by CKKH — APPROVED** [review]
Great work.

Goodbye py_boy 👋

**Review by hotblack86 — APPROVED** [review]
Yes! Awesome work guys.

File: src/game.py [hunk 1]
```diff
@@ -27,10 +27,12 @@ def __init__(self):
         self.enemy_sprites = pg.sprite.Group()
         self.hero = None
         self.enemy = None
+        self.win = None
         self.screen = pg.display.set_mode((WIDTH, HEIGHT))
         self.playing = None
         self.map_data = []
-        self.load_data()
+        self.map_list = ['map1.txt', 'map2.txt', 'map3.txt', 'map4.txt']
+        self.map_nr = 0
```
Comment (michaelnguyen974) [inline src/game.py:35]: Tasty

File: src/game.py [hunk 2]
```diff
@@ -157,6 +163,7 @@ def wait_for_key(self):
                     waiting = False
                     self.playing = False
                     pg.quit()
-                if event.type == pg.KEYUP:
+                if event.type == pg.KEYDOWN:
```
Comment (michaelnguyen974) [inline src/game.py:166]: Clicking down will make a new game

**Review by michaelnguyen974 — APPROVED** [review]

**Pull request merged**